src/js/reducers/projectDetailsReducer.js

```diff
@@ -202,8 +218,19 @@ export const getBookId = state => {
 export const getManifest = (state) =>
   state.manifest;
 
-export const getToolCategories = (state, toolName) =>
-[...state.toolsCategories[toolName]];
+/**
+ * Returns the categories selected for the tool
+ * @param state
+ * @param toolName
+ * @returns {*}
+ */
+export const getToolCategories = (state, toolName) => {
```

Comment:
@RoyalSix here's what happened. 

This was the original reducer:
```js
export const getToolCategories = (state, toolName) =>
[...state.toolsCategories[toolName]];
```

The problem ended up being that the wA tool doesn't actually have any categories so nothing would be written to the reducer which would give us a missing key error. This would break the project loading action and leave the project in a bad state.

This update ensures the tool has any categories first and falls back to a default empty array if not.

Reply:
Ok sounds good. Testing right now